maestro-cli/src/main/java/maestro/cli/cloud/CloudInteractor.kt

```diff
@@ -172,7 +172,7 @@ class CloudInteractor(
 
         val platform = Platform.fromString(deviceInfo.platform)
 
-        val line1 = "Maestro Cloud device specs:\n* ${deviceInfo.displayInfo}"
+        val line1 = "Maestro Cloud device specs:\n* ${deviceInfo.displayInfo} - locale ${deviceInfo.deviceLocale}"
         val line2 = "To change OS version use this option: ${if (platform == Platform.IOS) "--ios-version=<version>" else "--android-api-level=<version>"}"
```

Comment:
Can we also add a message with instructions on how to change locale like how we do for OS version?